testdata/error/message/invalid_duration.golden

```diff
@@ -0,0 +1 @@
+time: unknown unit  hour in duration 1 hour accessing 'timeout' (source:'test.source')
```

Comment:
This golden file has become obsolete. Maybe -update shall delete old golden files before (another PR)